### PR TITLE
Create AAZ commands and configs.

### PR DESCRIPTION
**Pull request by ajtiwari07**
Azure Cache for Redis new api v2023-04-01. 

- Used the AAZ Development Tool to create AAZ files.

### CODE REVIEW

**Comment by kairu-ms** [comment]
@ajtiwari07, could you resolve the conflicts in PR?

**Review by kairu-ms — APPROVED** [review]

**Comment by kairu-ms** [comment]
@ajtiwari07, it seems the varification failed. Please export your models from workspace directly into the latest main.

File: Commands/readme.md [hunk 1]
```diff
@@ -26,8 +26,8 @@
 - [billing-benefits](/Commands/billing-benefits/readme.md)
 : Azure billing benefits commands
 
-- [blueprint](/Commands/blueprint/readme.md)
-: Commands to manage blueprint.
```
Comment (necusjz) [inline Commands/readme.md:30]: why `blueprint` been removed here?

Comment (ajtiwari07) [reply Commands/readme.md]: Removed the unintentional change.

**Pull request merged**